### PR TITLE
fix for #1311 - Plos: Needs a facelift -- Move towards canonical Text tile

### PR DESCRIPTION
**Pull request by laouji**
Hi. This is a patch for the following issue: duckduckgo/zeroclickinfo-spice#1311

**Changes made:**
- [x] Changed layout from base template to text tile template (using mostly default styles)
- [x] Added author, journal name and issue number to instant answer

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)

plos_instantanswer

## Checklist

```
[x] Added metadata and attribution information
[] Wrote test file and added to t/ directory
[x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[x] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [x] Google Chrome
        [x] Firefox
        [] Opera
        [x] Safari

    - iOS (iPhone)
        [x] Safari Mobile
        [x] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [x] Google Chrome
        [] Opera
```

### CODE REVIEW

File: share/spice/plos/plos.js [hunk 1]
```diff
@@ -28,18 +28,34 @@
...
-		};
-	    },
+            normalize: function(item) {
+                var authors = item.author_display || [];
+                var subtitle = authors.length ? authors.join(", ") : "";
+
+                //strip html from title
+                var tmp = document.createElement('DIV');
+                tmp.innerHTML = item.title_display;
+                var title = tmp.textContent || tmp.innerText || '';
```
Comment (MrChrisW) [inline share/spice/plos/plos.js]: You can use the helper function `DDG.strip_html` to do this.

File: share/spice/plos/plos.js [hunk 2]
```diff
@@ -28,18 +28,34 @@
...
+
+                var journalName = item.journal ? item.journal + ' ' : '';
+                var issueNumber = item.issue ? 'Issue ' + item.issue : '';
```
Comment (MrChrisW) [inline share/spice/plos/plos.js]: Can you chain multiple var declarations as suggested in the [code style guide](https://duck.co/duckduckhack/code_styleguide)

**Comment by MrChrisW** [comment]
@laouji Thanks for taking this on! This is a great improvement.  I've added some feedback, let me know if you have any questions.

As the publication date is important it might be better to display it including the journal name like  
**2009 · PLoS ONE Issue 12**

As currently the date cannot be seen on some journals. 
selection_066

**Comment by laouji** [comment]
@MrChrisW Thanks for the helpful comments! I made some changes according to your suggestions.

The footer now looks like this:

plos_instantanswer2

**Comment by MrChrisW** [comment]
@laouji Happy to help :smiley: Thanks for updating with the suggested changes. 

I'm still not sure about the titles being cut off however this may be necessary as they're *quite* long! Just need to await suggestions from other community members and this should be good to merge! :shipit: 

CC @jagtalon @moollaza

**Comment by laouji** [comment]
Ok no problem.

I tried a couple other things like making the journal name or the authors the title, or letting the title take up two rows, but nothing really looked right to me.

If you have any ideas I'm willing to experiment some more.

**Comment by jagtalon** [comment]
:+1:

**Comment by jagtalon** [comment]
@laouji it's live! https://duckduckgo.com/?q=plos+dinosaurs&ia=research

**Comment by jkanarek** [comment]
@abeyang do you think that showing the plos logo on each tile is necessary?

**Comment by abeyang** [comment]
@jkanarek agreed. There are a few more changes I'd like to tweak as well.. will create a separate issue for them. /cc @jagtalon

**Pull request merged**